robottelo/ui/rhai.py

```diff
@@ -28,14 +29,18 @@ def register_client_to_rhai(self, activation_key, org):
 
         # Red Hat Access Insights requires RHEL 6/7 repo and it not
         # possible to sync the repo during the tests, Adding repo file.
-        self.vm.configure_rhel_repo(conf.properties['clients.rhel6_repo'])
-
-        try:
+        if rhel_distro == 'rhel67':
+            rhel_repo = conf.properties['clients.rhel6_repo']
```

Comment:
The config is the same, but now if that is not present you will not just skip it will raise a key error. Try commenting that values on the settings. The point here is, we should make this configuration required or not. If optional then we need to make sure that this method will work in the two cases.